frontend/public/components/factory/list-page.jsx

```diff
@@ -302,6 +306,7 @@ export const ListPage = withFallback(props => {
     name,
     namespace,
     selector,
+    hideFiltersIfEmpty = false,
```

Comment:
We don't want this to be the default for all other resources?

Reply:
+1, I think we should be consistent on all pages.